src/index.ts

```diff
@@ -363,6 +363,31 @@ export interface NumberC extends NumberType {}
  */
 export const number: NumberC = new NumberType()
 
+/**
+ * @since 2.0.7
+ */
+export class BigIntType extends Type<bigint> {
+  readonly _tag: 'BigIntType' = 'BigIntType'
+  constructor() {
+    super(
+      'bigint',
+      (u): u is bigint => typeof u === 'bigint',
```

Comment:
you can disable the offending rule with `// tslint:disable-next-line: valid-typeof` for now